pkg/controller/component/component_controller.go

```diff
@@ -282,3 +299,56 @@ func generateBuildConfig(namespace string, name string, builder *builderImage, g
 		},
 	}
 }
+
+func generateDeploymentConfig(namespace string, name string) *v1.DeploymentConfig {
+	labels := map[string]string{
+		"app": name,
+	}
+	return &v1.DeploymentConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name + "-dc",
```

Comment:
Do we really need the suffix here?

Reply:
Reason I ask is: oc new-app probably doesn't do it.

Reply:
done as part of f1e940c